pkgs/by-name/li/libzeep/package.nix

```diff
@@ -0,0 +1,35 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  boost,
+  date,
+  mrc,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "libzeep";
+  version = "6.0.13";
+
+  src = fetchFromGitHub {
+    owner = "mhekkel";
+    repo = "libzeep";
+    rev = "refs/tags/v${finalAttrs.version}";
+    hash = "sha256-74mYjz/jXzEU3j1kHSvTvZcIfX9csVJKTKONAZXUMvs=";
+  };
+
+  prePatch = ''
+    substituteInPlace CMakeLists.txt \
+      --replace-fail "3.0.1 QUIET" ""
```

Comment:
Likewise here. `sed` and a regex are likely to require less maintenance and allow for `version` and `src` to be overwritten with a newer version.